pathway/static/js/pathway/controllers/modal_pathway_ctrl.js

```diff
@@ -18,6 +20,14 @@ angular.module('opal.controllers').controller('ModalPathwayCtrl', function(
     );
     $scope.pathway = new pathwayService(pathwayDefinition, episode);
     $scope.editing = $scope.pathway.populateEditingDict(episode);
+    var analyticsKwargs = {
+      category: "Pathway"
+    }
```

Comment:
I think this should probably be ModalPathway - it would confuse me to see this in GA, I would think it were talking about page pathways...